openedx/core/lib/block_structure/tests/helpers.py

```diff
@@ -78,13 +78,15 @@ def __init__(self):
         # An in-memory map of cache keys to cache values.
         self.map = {}
         self.set_call_count = 0
+        self.timeout = 0
 
-    def set(self, key, val):
+    def set(self, key, val, timeout):
         """
         Associates the given key with the given value in the cache.
         """
         self.set_call_count += 1
         self.map[key] = val
+        self.timeout = timeout
```

Comment:
This is a Stub implementation of the django Cache.  It's similar to how the other Mock classes function in that they store information from past calls.  Would it be clearer if I rename this to `timeout_from_last_call` instead of `timeout`?